src/service.c

```diff
@@ -828,7 +834,7 @@ create_logout_section (IndicatorSessionService * self)
 
   menu = g_menu_new ();
 
-  if (indicator_session_actions_can_logout (p->backend_actions))
+  if (self->priv->bLocal && indicator_session_actions_can_logout (p->backend_actions))
```

Comment:
You can also logout from a remote session. Please leave the logout menu item when connecting from remote.